manim/mobject/types/image_mobject.py

```diff
@@ -66,41 +66,16 @@ def set_interpolation_algorithm(self, interpolation_algorithm):
          * 'lanczos' or 'antialias'
         The string constants can be input either in lowercase or in UPPERCASE, like the corresponding constant name in the Pillow library.
         """
-        if isinstance(interpolation_algorithm, str):
-            if (
-                interpolation_algorithm.lower() == "bicubic"
-                or interpolation_algorithm.lower() == "cubic"
-            ):
-                interpolation_algorithm = Image.BICUBIC
-            elif (
-                interpolation_algorithm.lower() == "nearest"
-                or interpolation_algorithm.lower() == "none"
-            ):
-                interpolation_algorithm = Image.NEAREST
-            elif interpolation_algorithm.lower() == "box":
-                interpolation_algorithm = Image.BOX
-            elif (
-                interpolation_algorithm.lower() == "bilinear"
-                or interpolation_algorithm.lower() == "linear"
-            ):
-                interpolation_algorithm = Image.BILINEAR
-            elif interpolation_algorithm.lower() == "hamming":
-                interpolation_algorithm = Image.HAMMING
-            elif (
-                interpolation_algorithm.lower() == "lanczos"
-                or interpolation_algorithm.lower() == "antialias"
-            ):
-                interpolation_algorithm = Image.LANCZOS
-            else:
-                raise ValueError(
-                    f"interpolation_algorithm has a wrong value: '{interpolation_algorithm}'. It must be one of 'bicubic', 'nearest', 'box', 'bilinear', 'hamming', 'lanczos' (case insensitive) or a Pillow resampling filter integer constant."
+        if isinstance(resampling_algorithm, str):
+            raise ValueError(
+                    f"resampling_algorithm cannot be string. RESAMPLING_ALGORITHMS[\"<algorithm>\"] or Pillow resampling filter constants. Available algorithms: 'bicubic', 'nearest', 'box', 'bilinear', 'hamming', 'lanczos'."
                 )
-            self.interpolation_algorithm = interpolation_algorithm
-        elif isinstance(interpolation_algorithm, int):
-            self.interpolation_algorithm = interpolation_algorithm
+            self.resampling_algorithm = resampling_algorithm
```

Comment:
Maybe it's the github diff that is misleading me, but what is the line 73? Given that it is right after the ValueError at line 70, it will never be reached.

Reply:
@huguesdevimeux  Good find. It is an artefact of my old code.